nav2_smac_planner/src/a_star.cpp

```diff
@@ -572,7 +572,9 @@ typename AStarAlgorithm<NodeT>::AnalyticExpansionNodes AStarAlgorithm<NodeT>::ge
   for (float i = 1; i < num_intervals; i++) {
     node->motion_table.state_space->interpolate(from(), to(), i / num_intervals, s());
     reals = s.reals();
-    angle = reals[2] / node->motion_table.bin_size;
+    //TODO: Bin size removed from motion table for state lattice
+    // angle = reals[2] / node->motion_table.bin_size;
+    angle = 0.0;
```

Comment:
This needs to be converted from 0-2*pi (or maybe -pi to pi?) into an angular bin number to test collisions for. Basically, it needs the opposite of the getRadFromBin and instead getBinFromRad